code/parachain/runtime/picasso/src/migrations.rs

```diff
@@ -45,6 +45,67 @@ impl OnRuntimeUpgrade for TechCollectiveRenameMigration {
 	}
 }
 
+mod pablo_picasso_init_pools {
+
+	use super::*;
+
+	use frame_support::BoundedBTreeMap;
+	use pablo::pallet::PoolInitConfiguration;
+	use sp_runtime::PerThing;
+
+	pub struct PabloPicassoInitialPoolsMigration;
+
+	fn add_initial_pools_to_storage() -> Weight {
+		Pablo::do_create_pool(
+			create_two_token_pool_config(
+				CurrencyId::KSM,
+				CurrencyId::USDT,
+				Permill::from_percent(50),
+			),
+			Some(CurrencyId::KSM_USDT_LPT),
+		)
+		.expect("Is a valid pool config; QED");
+
+		Pablo::do_create_pool(
+			create_two_token_pool_config(
+				CurrencyId::PICA,
+				CurrencyId::USDT,
+				Permill::from_percent(50),
+			),
+			Some(CurrencyId::PICA_USDT_LPT),
+		)
+		.expect("Is a valid pool config; QED");
+		// TODO(connor) calculate weight based of read/write count
```

Comment:
it should be possible to write a benchmark for `do_create_pool` when passing an LP token in